docs/adding_tracks.rst

```diff
@@ -611,15 +611,15 @@ Sharing your track with others
 
 So far the track is only available on your local machine. To share your track you could check it into version control. To avoid committing the potentially huge data file you can expose it via http (e.g. via an S3 bucket) and reference it in your track with the property ``base-url``. Rally expects that the URL points to the parent path and appends the document file name automatically.
 
-You should also compress your document corpus to save network bandwidth; bzip2 works well. You can create a compressed archive with the following command::
+You should also compress your document corpus to save network bandwidth; `pbzip2 <https://linux.die.net/man/1/pbzip2>`_ works well, is backwards compatible with ``bzip2`` and makes use of all available cpu cores for compression and decompression. You can create a compressed archive with the following command::
 
-    bzip2 -9 -c documents.json > documents.json.bz2
+    pbzip2 -9 -k -m2000 -v documents.json
```

Comment:
`2GB` turns out to be the maximum size (for compression); note this is a hint on the max memory it could gobble up, provided it's available.